checker/tests/nullness/flow/Issue3249.java

```diff
@@ -0,0 +1,13 @@
+public class Issue3249 {
+
+    private final double field;
+
+    Issue3249() {
+        double local;
+        while (true) {
```

Comment:
Didn't you have more tests with more complicated expressions, like `!false`, `true || foo` etc?
Can you add a few to show what javac handles?

Reply:
This test case is only applicable to constructor, since the final field should be initialized in constructor. Or should I add more separate test files?